recipes/oauth2-proxy/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set provider = "oauth2-proxy" %}
+{% set goname = "github.com/oauth2-proxy/"+provider.lower() %}
+{% set version = "5.1.1" %}
+
+{% set name = goname.split('/')[-1] %}
+{% set pkg_src = ('src/'+goname).replace("/",os.sep) %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - folder: {{ pkg_src }}
+    url: https://{{ goname }}/archive/v{{ version }}.tar.gz
+    sha256: f87b9596420739328e9271ec51c092190039521f4e1daf552123ded27b635def
+
+build:
+  number: 0
+  script:
+    - pushd {{ pkg_src }}
+    - GOBIN=$(go env GOBIN)  # [not win]
+    - FOR /F "delims=" %%i IN ('go env GOBIN') DO set "GOBIN=%%i"  # [win]
+    - go build -v -o $GOBIN/{{ name }} .       # [unix]
+    - go build -v -o %GOBIN%\{{ name }}.exe .  # [win]
+
+requirements:
+  build:
+    - {{ compiler('go') }}
+
+test:
+  requires:
+    - m2-bash       # [win]
+    - m2-coreutils  # [win]
+
+  commands:
+    - oauth2-proxy --version
+
+about:
+  home: https://github.com/oauth2-proxy/{{ name }}
+  license: MIT
+  license_file: {{ pkg_src }}/LICENSE
+  summary: A reverse proxy and static file server that provides authentication using Providers (Google, GitHub, and others) to validate accounts by email, domain or group.
+  doc_url: https://oauth2-proxy.github.io/oauth2-proxy/
+  dev_url: https://github.com/oauth2-proxy/{{ name }}
+
+extra:
+  recipe-maintainers:
+    - rluria14
+    - ndmaxar
+    - oblute
+    - RyanHackney
```

Comment:
@RyanHackney, are you happy to be listed here as a maintainer?

Reply:
@chrisburr , yes, happy to be listed as a maintainer